test/api-digester/source-stability.swift.expected

```diff
@@ -17,27 +13,176 @@ Protocol MutableIndexable has been removed (deprecated)
 Protocol RandomAccessIndexable has been removed (deprecated)
 Protocol RangeReplaceableIndexable has been removed (deprecated)
 
-/* More DictionaryIndex / SetIndex */
+/* FIXME: Bogus */
 Var Dictionary.endIndex has declared type change from DictionaryIndex<Key, Value> to Dictionary<Key, Value>.Index
 Var Dictionary.startIndex has declared type change from DictionaryIndex<Key, Value> to Dictionary<Key, Value>.Index
 Var Set.endIndex has declared type change from SetIndex<Element> to Set<Element>.Index
 Var Set.startIndex has declared type change from SetIndex<Element> to Set<Element>.Index
+Constructor BidirectionalSlice.init(base:bounds:) has 2nd parameter type change from Range<Base.Index> to Range<BidirectionalSlice.Index>
```

Comment:
@nkcsgexi I'm really sorry for constantly breaking the output of this tool, but we need to figure out a way to canonicalize types better.

In particular, it seems we need a better strategy for handling changes where the serialized type resolves to an alias of the same type as before, or a generic parameter that is related to the previous one by a same-type constraint. I'm at a loss as to how to work this into the model of "dump API of two serialized module files, diff them". We might need a more complex approach.

Or, since with this patch the interface type changeover is mostly complete, we can start again with a fresh dump for the "known good" API.